carbon-design-system/carbon
# fix: inline dropdown has border bottom in readonly variant

Closes #15367 

fixed -  inline dropdown has border bottom in readonly variant

#### Testing / Reviewing

1. Go to Dropdown
2. Change its type as 'inline'
3. Change 'readonly' as true
4. It should not show a border bottom
